ContributedExamples/my_first_notebook.ipynb

```diff
@@ -0,0 +1,128 @@
+{
```

Comment:
<p>This cell looks wrong...</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/COSIMA/cosima-recipes/pull/210/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>